visualization/app/codeCharta/state/store/appSettings/appSettings.splitter.ts

```diff
@@ -1,136 +1,141 @@
+import {splitCameraTargetAction} from "./cameraTarget/cameraTarget.splitter"
+import {splitIsAttributeSideBarVisibleAction} from "./isAttributeSideBarVisible/isAttributeSideBarVisible.splitter"
+import {splitSortingOrderAscendingAction} from "./sortingOrderAscending/sortingOrderAscending.splitter"
+import {splitSearchPanelModeAction} from "./searchPanelMode/searchPanelMode.splitter"
+import {splitIsLoadingFileAction} from "./isLoadingFile/isLoadingFile.splitter"
+import {splitIsLoadingMapAction} from "./isLoadingMap/isLoadingMap.splitter"
+import {splitMapColorsAction} from "./mapColors/mapColors.splitter"
+import {splitResetCameraIfNewFileIsLoadedAction} from "./resetCameraIfNewFileIsLoaded/resetCameraIfNewFileIsLoaded.splitter"
+import {splitShowOnlyBuildingsWithEdgesAction} from "./showOnlyBuildingsWithEdges/showOnlyBuildingsWithEdges.splitter"
+import {splitWhiteColorBuildingsAction} from "./whiteColorBuildings/whiteColorBuildings.splitter"
+import {splitIsWhiteBackgroundAction} from "./isWhiteBackground/isWhiteBackground.splitter"
+import {splitDynamicMarginAction} from "./dynamicMargin/dynamicMargin.splitter"
+import {splitInvertHeightAction} from "./invertHeight/invertHeight.splitter"
+import {splitInvertDeltaColorsAction} from "./invertDeltaColors/invertDeltaColors.splitter"
+import {splitInvertColorRangeAction} from "./invertColorRange/invertColorRange.splitter"
+import {splitHideFlatBuildingsAction} from "./hideFlatBuildings/hideFlatBuildings.splitter"
+import {splitCameraAction} from "./camera/camera.splitter"
+import {splitScalingAction} from "./scaling/scaling.splitter"
+import {splitEdgeHeightAction} from "./edgeHeight/edgeHeight.splitter"
+import {splitAmountOfEdgePreviewsAction} from "./amountOfEdgePreviews/amountOfEdgePreviews.splitter"
+import {splitAmountOfTopLabelsAction} from "./amountOfTopLabels/amountOfTopLabels.splitter"
+import {splitIsPresentationModeAction} from "./isPresentationMode/isPresentationMode.splitter"
+import {splitExperimentalFeaturesEnabledAction} from "./enableExperimentalFeatures/experimentalFeaturesEnabled.splitter"
 
 export function splitAppSettingsActions(payload: RecursivePartial<AppSettings>) {
-	const actions: CCAction[] = []
+    const actions: CCAction[] = []
 
-	// Plop: Append action split here
-	if (payload.showMetricLabelNodeName !== undefined) {
-		actions.push(splitShowMetricLabelNodeNameAction(payload.showMetricLabelNodeName))
-	}
+    // Plop: Append action split here
+    if (payload.showMetricLabelNodeName !== undefined) {
+        actions.push(splitShowMetricLabelNodeNameAction(payload.showMetricLabelNodeName))
+    }
 
-	if (payload.showMetricLabelNameValue !== undefined) {
-		actions.push(splitShowMetricLabelNameValueAction(payload.showMetricLabelNameValue))
-	}
+    if (payload.showMetricLabelNameValue !== undefined) {
+        actions.push(splitShowMetricLabelNameValueAction(payload.showMetricLabelNameValue))
+    }
```

Comment:
The spacing seem to have changed? 
Are you using the commit pre-hooks, which should set those as far as I know, or the IDEA code-format option?

Reply:
Those are an examplary line choice, the spacing is changed in the whole file.